@pwablito The `slurm` package builds, but the `slurm-drmaa` doesn't when compiled in a clean Ubuntu 18.04 container:
```
     107    checking for usable SLURM libraries/headers... *** The SLURM test program failed to link 
            or run. See the file config.log
     108    *** for the exact error that occured.
     109    no
  >> 110    configure: error:
     111    SLURM libraries/headers not found;
     112    add --with-slurm-inc and --with-slurm-lib with appropriate locations.
```
config.log
```
configure:13714: checking for usable SLURM libraries/headers
configure:13735: /home/kaindlb/spack/spack/lib/spack/env/gcc/gcc -o conftest -pedantic -std=c99 -g -O2
 -pthread -D_REENTRANT -D_THREAD_SAFE -DNDEBUG  -D_GNU_SOURCE -I/home/kaindlb/spack/spack/opt/spack/li
nux-ubuntu18.04-zen/gcc-8.4.0/slurm-21-08-1-1-6jbxvnvk5douefrouygeavqcetcvpw5z/include  -L/home/kaindl
b/spack/spack/opt/spack/linux-ubuntu18.04-zen/gcc-8.4.0/slurm-21-08-1-1-6jbxvnvk5douefrouygeavqcetcvpw
5z/lib conftest.c -pthread  -lslurmdb -lslurm  >&5
/usr/bin/ld: cannot find -lslurmdb
```
found in opt that libslurm is built, libslurmdb not.

--- a/var/spack/repos/builtin/packages/slurm-drmaa/package.py
+++ b/var/spack/repos/builtin/packages/slurm-drmaa/package.py
@@ -16,15 +16,27 @@ class SlurmDrmaa(AutotoolsPackage):
     '''
     homepage = "https://github.com/natefoo/slurm-drmaa"
     url      = "https://github.com/natefoo/slurm-drmaa/releases/download/1.1.2/slurm-drmaa-1.1.2.tar.gz"
+    git      = 'https://github.com/natefoo/slurm-drmaa.git'
 
     maintainers = ['pwablito']
 
-    version('1.2.0-dev.635b7ac', sha256='5aaf12e87010315ebf5217fe8a21003ab967c42b74efea33243dff221c85c2dc')
+    version('main', branch='main',  submodules=True)
+    version('1.1.2', sha256='5bfe25d501de83729df3c8c8f28535b9da3e99aea7738e259903abd6f1f5c836')
+
+    # Remove this patch when it is merged into main:
+    patch('https://patch-diff.githubusercontent.com/raw/natefoo/slurm-drmaa/pull/62.patch',
+          sha256='86bba5e2683f33e59f7e9193aa5fa948eb6a5507d44768f2f6d756a33fef7b74',
+          when='@main')
+
+    depends_on('autoconf', type='build', when='@main')
+    depends_on('automake', type='build', when='@main')
+    depends_on('libtool',  type='build', when='@main')
+    depends_on('bison',    type='build', when='@main')
 
     depends_on("slurm")
+    depends_on("slurm@:20-11-8-1", when='@1.1.2')
     depends_on("gperf")
     depends_on("ragel")
 
-    def configure_args(self):
-        args = []
-        return args
+    def check(self):
+        pass

